tools/cooja/apps/avrora/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: "java"
+
+jar.baseName = 'cooja_avrora'
+
+repositories {
+  jcenter()
+}
+
+sourceSets {
+  main {
+    java {
+      srcDir 'src'
+    }
+  }
+
+}
+
+
+dependencies {
+    compile project(":")
+    compile files("lib/avrora-small.jar")
```

Comment:
Yea, I'm well aware that it does does not have a circular _build_ dependency. I just noticed that a "want _to run_ Cooja without the plugins." would have been a better wording.

But back to the topic at hand: To explain my issues in more detail:
You have two local git-checkouts of Cooja - Co1 and Co2. You compile and install the Cooja package in Co1. Now you compile a plugin in Co2. Due to the repo-magic it will use the Cooja-build from Co1 to build against as it as matching is done one name+version and both artifacts will have the same version, as Cooja has no versioning. Even if building does not fail this can cause problems when building Cooja in Co2 afterwards when the class-loader loads that plugin. This is for example the case when you add a function to one of Cooja's classes in Co2 that the is not used by that plugin, yet the class itself is referenced. 

So if you suggest manually setting the repo-path to the, again relative, path 'tools/.m2' or similar that is ok for me, yet I don't see any advantage over manually referencing those files, which is much easier to understand for people not familiar with the maven-repo thing.